resource-definitions/generic-async-driver/job-pod-template-source-files/agent-runner.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: entity.humanitec.io/v1b1
```

Comment:
Our Kapa.ai team has advised us to add a summary line at the top of code examples so that the AI gets a hint at what this piece of code does. Here's a suggestion:
```suggestion
# This Resource Definition specifies the Humanitec Agent to use for the Runner.
apiVersion: entity.humanitec.io/v1b1
```